Index: passwords.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/passwords.py b/passwords.py
new file mode 100644
--- /dev/null	(date 1640568761188)
+++ b/passwords.py	(date 1640568761188)
@@ -0,0 +1,4 @@
+mongoDB_atlas = {
+    'username': 'irineu',
+    'password': 'zUIPb5fSgFth7Vlg'
+}
\ No newline at end of file
